<?xml version="1.0"?>

<launch>
    <!-- Robot namespace -->
    <arg name="robot_name" default="my_gen3"/>
    <arg name="pose_input" default="/pcl_closest_obj/output_pose"/>
    <arg name="start_delay_seconds" default="0"/>

    <!-- Launch the example in that namespace -->
    <!-- node name="full_arm_movement_python" pkg="robothon2023" type="full_arm_movement.py" output="screen" launch-prefix="bash -c 'sleep $(arg start_delay_seconds); $0 $@' ">
        <param name="robot_name" value="$(arg robot_name)"/>
    </node -->

    <node pkg="robothon2023" type="slider_test.py" name="slider_test" output="screen">

    </node>
    <node pkg="tf2_ros" type="static_transform_publisher" name="board_link_pub" args="0.4688 0.3703 0.069 0.0 0.0 -0.765 0.643 base_link board_link" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="slider_start_pub" args="0.057 -0.035 0 0 0 0 1 board_link slider_start_link" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="slider_end_pub" args="0.045 0 0 0 0 0 1 slider_start_link slider_end_link" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="meter_plug_black_pub" args="0 -0.058 0 0 0 0 1 board_link meter_plug_black_link" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="meter_plug_red_pub" args="0.025 0 0 0 0 0 1 meter_plug_black_link meter_plug_red_link" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="door_knob_pub" args="0.005 -0.142 0 0 0 0 1 board_link door_knob_link" />
</launch>
